### PR TITLE
AB#32465: Bulk Submissions Settings View 

### PR DESCRIPTION
**Pull request by shakisparki**
## Overview

Add Biobank Bulk Submissions Settings View

### CODE REVIEW

**Review by beforan — DISMISSED** [review]
I messed up and blocked the wrong PR.

This PR blocks #188 until its merged

**Comment by beforan** [comment]
@shakisparki are migrations resolved yet after #187 merge?

**Comment by shakisparki** [comment]
> @shakisparki are migrations resolved yet after #187 merge?

Yes, all resolved for this PR

File: src/Directory/Biobanks.Web/Views/Biobank/Submissions.cshtml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+@Html.Partial("_BiobankTabs", "Submissions")
+
+<h2>
+    @ViewBag.Title
+</h2>
+
+<p>
+    If you need to bulk submit data, instructions on how to interact with the submissions API can be
+    found here (todo link).
```
Comment (Flemingjp) [inline src/Directory/Biobanks.Web/Views/Biobank/Submissions.cshtml:15]: (todo link)

File: src/Directory/Biobanks.Web/Views/Biobank/Submissions.cshtml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                }
+            </div>
+            @Html.ValidationMessageFor(x => x.AccessCondition)
+        </div>
+    </div>
+
+    <div class="form-group">
+        <div class="col-sm-10 col-sm-offset-1">
+            <h4>
+                @Html.DisplayNameFor(x => x.CollectionType)
```
Comment (Flemingjp) [inline src/Directory/Biobanks.Web/Views/Biobank/Submissions.cshtml:60]: Appears to be a required field - needs the required class

File: src/Entities/Data/Organisation.cs [hunk 3]
```diff
@@ -58,6 +59,17 @@ public class Organisation
...
+
+        public int? DefaultSubmissionsCollectionTypeId { get; set; }
+
+
+        [ForeignKey("DefaultSubmissionsAccessConditionId")]
+        public virtual AccessCondition AccessCondition { get; set; }
+
+        [ForeignKey("DefaultSubmissionsCollectionTypeId")]
+        public virtual CollectionType CollectionType { get; set; }
+
```
Comment (Flemingjp) [inline src/Entities/Data/Organisation.cs:72]: Current convention is to have a ID field for the FK - `AccessConditionId` for example. Very few entities uses the DataAnnotations.

Am I correct in my thinking @blargly?

**Review by Flemingjp — CHANGES_REQUESTED** [review]
- Missing an entry for the `Mvc.sitemap`, this means the navigation bar is missing on the page

File: src/Entities/Data/Organisation.cs (hunk 3 above)
Comment (Flemingjp) [reply src/Entities/Data/Organisation.cs:72]: Unless the Ids are used - they can be dropped, along with the ForeignKey annotation. EF should be able to populate `AccessConition` and `CollectionType` from the type alone

**Review by blargly — APPROVED** [review]

**Review by blargly — DISMISSED** [review]

File: src/Directory/Biobanks.Web/Views/Biobank/Submissions.cshtml (hunk 1 above)
Comment (beforan) [reply src/Directory/Biobanks.Web/Views/Biobank/Submissions.cshtml:15]: if the a link is available (I suspect not) then obvs add it.

If not then:
- I'm ok with this `<p>` being commented out, with a `TODO` in the comments.
- Create a work item in DevOps for getting the link and adding it to the view
- The `TODO` should reference the work item ID

**Review by beforan — COMMENTED** [review]
just clearing my previous changes requested so I'm not a blocker ;)

**Pull request closed without merging**